doc/source/cluster/config.rst

```diff
@@ -341,7 +341,7 @@ The key is the name of the node type, which is just for debugging purposes.
                 resources: {"CPU": 2}
                 min_workers: 0
                 max_workers: 0
-            ray.worker.small:
+            ray.worker.default:
```

Comment:
line 351 below should be `max_workers: 2` right?

Reply:
indeed